modules/5_install/install.tf

```diff
@@ -240,10 +240,8 @@ firewall-cmd --zone=public --add-masquerade --permanent
 # Masquerade will enable ip forwarding automatically
 firewall-cmd --reload
 
-ethtool -K $PUBLIC_INTERFACE tso off
-ethtool -K $PUBLIC_INTERFACE gso off
-ethtool -K $PRIVATE_INTERFACE tso off
-ethtool -K $PRIVATE_INTERFACE gso off
+#Checksum needs to be turned off to avoid a bug with ibmveth
+ethtool -K $PRIVATE_INTERFACE rx off
```

Comment:
Is it not required for `PUBLIC_INTERFACE`? If not can we remove the line at 237? https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/6746cc11ce9558949b9bc1d9e60ebbeae745b972/modules/5_install/install.tf#L237

Reply:
Yes, we can remove it